framework/src/play/data/validation/ValidationPlugin.java

```diff
@@ -159,7 +159,7 @@ static void save() {
         if (Validation.errors().isEmpty()) {
             // Only send "delete cookie" header when the cookie was present in the request
             if(Http.Request.current().cookies.containsKey(Scope.COOKIE_PREFIX + "_ERRORS") || !Scope.SESSION_SEND_ONLY_IF_CHANGED) {
-                Http.Response.current().setCookie(Scope.COOKIE_PREFIX + "_ERRORS", "", null, "/", 0, Scope.COOKIE_SECURE, Scope.SESSION_HTTPONLY);
+                Http.Response.current().setCookie(Scope.COOKIE_PREFIX + "_ERRORS", "", null, "/", 0, Scope.COOKIE_SECURE, Scope.SESSION_HTTPONLY, Scope.SESSION_SAMESITE);
```

Comment:
Isn't it strange that we use SameSite setting which was introduced **for session cookie** also for some other cookies?

Reply:
Maybe, idk. What do you suggest? null / ""? 
It uses the same settings as session cookie for httpOnly and Secure, so I thought that it should also use same settings as session for samesite.

Reply:
Updated PR